partners/icr-scan/0.1/icr-scan.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: icr-scan
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.17.0"
+    tekton.dev/categories: Security
+    tekton.dev/tags: security
+    tekton.dev/displayName: "icr scanner"
+    tekton.dev/platforms: "linux/amd64"
+spec:
+  description: >-
+    The following task can be used to perform analysis on the source code
+    and send the results to the iCR server specified by the parameter Server-URL
+  params:
+
+    - name: iCR-License
+      description: The license required to use the iCR to scan the source code.
+
+    - name: Git-Revision
+      description: Branch of the Git Repository
+      default: master
+
+    - name: Server-URL
+      description: The URL of the Server where data will be sent
+
+    - name: iCR-Username
+      description: The Username for the iCR Server
+      default: admin
+
+    - name: iCR-Password
```

Comment:
isn't this a security flaw to pass password via parameters? Shouldn't ba password stored in Openshift Secrets and  only secret referenced here?

Reply:
BTW parameters appear in clear text on the image attestation

Reply:
This is just a placeholder. We expect that the users will use secrets instead of plaintext